defaultPane.js

```diff
@@ -5,14 +5,17 @@
 ** This pane hides the ones considered too low-level for the normal user.
 */
 
-var UI = require('solid-ui')
-var panes = require('pane-registry')
+const UI = require('solid-ui')
+const panes = require('pane-registry')
+const ns = UI.ns
 
 module.exports = {
   icon: UI.icons.originalIconBase + 'about.png',
 
   name: 'default',
 
+  audience: [ns.solid('Developer')],
```

Comment:
I'm not quite sure what this Pane does, but given that it's named "Default": is this the one that will be visible if no other Pane matches? In other words: will the user get an empty screen if this one's not present either? If so, maybe it should just do what it does today if the user's a Developer, and display a message about why the user doesn't see anything if not.

Reply:
I think the name is just misleading - I suspect it was one of the first panes added, and the idea was to give you a little bit of info about the resource. But this info is still designed for developers.

If no panes are found, it will show `internalPane` instead. (This is also noted as developer-oriented, but this is hard-coded to be shown if no other panes are available.)